services/validatorWorker/follower.js

```diff
@@ -1,5 +1,5 @@
 const BN = require('bn.js')
-const { getStateRootHash, onError, toBNMap } = require('./lib')
+const { getStateRootHash, onError, toBNMap, sumMap } = require('./lib')
```

Comment:
no need to have any changes in `follower.js` since `exhausted` property should be removed from `ApproveState`